models/pathology_tumor_detection/configs/metadata.json

```diff
@@ -5,11 +5,11 @@
         "0.1.0": "initialize release of the bundle"
     },
     "monai_version": "0.9.1",
-    "pytorch_version": "1.10.0",
+    "pytorch_version": "1.12.0",
     "numpy_version": "1.21.2",
+    "torchvision": "0.13.0",
+    "cucim": "22.04",
     "optional_packages_version": {
-        "cucim": "22.04",
```

Comment:
HI @drbeh , here could you put torchvision and cucim back into `optional_packages_version`?

Reply:
Sure, but can let me know why is that? I don't get the concept of optional here as they are required for this bundle.

Reply:
Hi @drbeh ,

I think we treat cucim and torchvision as optional dependencies of MONAI:
https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies
So here the "optional" means the same.

Thanks.